nilearn/regions/tests/test_signal_extraction.py

```diff
@@ -13,13 +13,20 @@
 from nilearn._utils.testing import write_tmp_imgs
 from nilearn._utils.data_gen import generate_timeseries, generate_regions_ts
 from nilearn._utils.data_gen import generate_labeled_regions, generate_maps
+from nilearn._utils.data_gen import generate_labeled_regions_large
 from nilearn._utils.data_gen import generate_fake_fmri
 from nilearn._utils.exceptions import DimensionError
 from nilearn.image import get_data, new_img_like
 
 _TEST_DIM_ERROR_MSG = ("Input data has incompatible dimensionality: "
                        "Expected dimension is 3D and you provided "
                        "a 4D image")
+_TEST_SHAPE_LABEL_ERROR_MSG = "label img and imgs shapes must be identical."
```

Comment:
These need to be updated to match the error messages in `signal_extraction.py (`label_imgs.shape and target_shape must be identical.` for example).